app/liquid_tags/github_tag/github_readme_tag.rb

```diff
@@ -3,71 +3,79 @@
 class GithubTag
   class GithubReadmeTag
     PARTIAL = "liquids/github_readme".freeze
+    GITHUB_DOMAIN_REGEXP = %r{.*github.com/}.freeze
+    OPTION_NO_README = "no-readme".freeze
+    VALID_OPTIONS = [OPTION_NO_README].freeze
```

Comment:
I'm confused about how the readme plays into this. Can you explain how that works? The `show_readme?` being true when `options.none?([OPTION_NO_README])` was kinda making my head spin with the negatives.

Reply:
yeah, that's how to the tag was implemented. By default it shows the readme, the option `no-readme` doesn't render one

```liquid
{% github https://github.com/rust-lang/rust %}
```

renders

![Screenshot_2020-05-07 New Post - DEV Community 👩‍💻👨‍💻](https://user-images.githubusercontent.com/146201/81304836-e1184f80-907d-11ea-93e6-9080c16fe4a7.png)

```liquid
{% github https://github.com/rust-lang/rust no-readme %}
```

renders

![Screenshot_2020-05-07 New Post - DEV Community 👩‍💻👨‍💻(1)](https://user-images.githubusercontent.com/146201/81304899-fbeac400-907d-11ea-8364-1e44e08b9101.png)

the `show_readme?` returns true unless `no-readme` is present in the options